### ISSUE TITLE
Option to select download quality

### ISSUE THREAD

**Issue by hardiktaluja** [arashnm80/spot-seek-bot#11]
Please add an option to select the download quality. The current default is 320kbps.

**Comment by kvfgb-py** [comment]
All of these bots first download the song in 128 kbps by spotdl and then by FFmpeg convert it to 320 kbps, this just make the song size larger and no effect on quality.

**Comment by kvfgb-py** [comment]
Mine download the song in 128 kbps 
[SpotFather](Https://t.me/SpotFather_Robot)

**Comment by leosamuele221** [comment]
True! there is really no need to download at 320kpbs if the source is 128kbps.
It should be written in the readme @arashnm80 stating that the bot downloads at 320kbps is a lie.

**Comment by arashnm80** [comment]
the 320kbps flag is passed to spotdl in download and the highest available quality is downloaded (which might differ from track to track)
